yeat/tests/data/configs/paired.cfg

```diff
@@ -33,6 +33,22 @@
                 "Shigella_sonnei_53G"
             ],
             "mode": "paired"
+        },
+        "penguin-default": {
+            "algorithm": "penguin",
+            "extra_args": "",
+            "samples": [
+                "Shigella_sonnei_53G"
+            ],
+            "mode": "paired"
+        },
```

Comment:
Test data config files for the `Penguin` assembler in #82 were not updated. Updating in this PR.